internal/server/middleware_test.go

```diff
@@ -373,8 +372,9 @@ func TestGetOrgFromRequest_FromRequestHost(t *testing.T) {
 	assert.Equal(t, resp.StatusCode, 200)
 }
 
-func TestGetOrgFromRequest_MissingOrgErrors(t *testing.T) {
+func TestGetOrgFromRequest_MissingOrgErrorsInMultiTenantEnvironment(t *testing.T) {
 	srv := setupServer(t, withAdminUser)
+	srv.options.EnableSignup = true // test multi-tenancy environment
```

Comment:
This test was not correctly testing the case we wanted because it did not use the `setOrganizationInCtx` middleware. Now that the middleware is integrated the test failed and required this fix.